lighthouse-core/gather/drivers/connection.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+const log = require('../../lib/log.js');
+
+class Connection {
+
+  constructor() {
+    this._lastCommandId = 0;
+    /** @type {!Map<number, function(object)}*/
```

Comment:
I think this is now `/** @type {!Map<number, {resolve: function(*), reject: function(*), method: string}>}*/`